### PR TITLE
[ambient] Add to the Graph Traffic Dropdown

### LINKED ISSUE #6900: Ambient - Visualize ztunnel in the graph
The most important thing would be to show or be able to filter when a graph is Ambient / Where the telemetry is coming from. 

With that information, it will be more easy for the use to understand the graph (TCP edges from ztunnel, etc)

In a namespace with sidecar, it really doesn't show the sidecar, but it shows when it is missing: 

image

It could be useful to have something indicating that the traffic of the workload is redirected throw ztunnel. Some ideas: 

- Different arrow color
- A new icon - Like the locker - next to the arrow. 
- Or maybe the locker itself but with a different color 
- A new node badge

### PR DESCRIPTION
**Pull request by jshaughn**
Closes #6900 

Add to the Graph Traffic Dropdown, allowing the user to select no ambient reporting, full ambient reporting, only waypoint or only ztunnel.  This should help users identify or remove ambient-reported traffic.

image

### CODE REVIEW

**Review by josunect — COMMENTED** [review]
I think this is going to be very helpful for users to be able to understand the graph with Ambient. 

I just have a couple of comments: 

- Should the Ambient option be available if we are not in Ambient mesh? Ambient options are hidden (At least for now) is the Mesh is not Ambient. 
image

- When the ztunnel is selected, It is possible to see nodes with no traffic, but "iddle nodes" are not selected, shouldn't they be hidden? 
image

**Comment by jshaughn** [comment]
> Should the Ambient option be available if we are not in Ambient mesh? Ambient options are hidden (At least for now) is the Mesh is not Ambient.

Good point, I will make the change!

> When the ztunnel is selected, It is possible to see nodes with no traffic, but "iddle nodes" are not selected, shouldn't they be hidden?

I think that may actually be a separate issue.  I think this may be in the appender.  When we remove waypoint nodes we should probably then clean up orphan nodes.  In your screenshot you have only ztunnel, but I assume you also are not showing waypoints?

As an aside, the traffic that IS being shown is reported via proxy, not ambient, because the gateway is a proxy.  This shows how you can remove ambient telem and still see proxy telem.

File: frontend/src/pages/Graph/GraphToolbar/GraphTraffic.tsx [hunk 1]
```diff
@@ -97,6 +111,35 @@ const GraphTrafficComponent: React.FC<GraphTrafficProps> = (props: GraphTrafficP
...
+        tooltip: (
+          <div style={{ textAlign: 'left' }}>Limit to only waypoint reported traffic, for the enabled protocols.</div>
+        )
+      },
+      {
+        id: TrafficRate.AMBIENT_ZTUNNEL,
+        labelText: 'ZTunnel',
+        isChecked: trafficRates.includes(TrafficRate.AMBIENT_ZTUNNEL),
+        tooltip: (
+          <div style={{ textAlign: 'left' }}>Limit to only ztunnel reported traffic, for the enabled protocols.</div>
```
Comment (josunect) [inline frontend/src/pages/Graph/GraphToolbar/GraphTraffic.tsx:128]: Could be helpful adding that ztunnel reports just at L4/TCP traffic?

**Review by josunect — COMMENTED** [review]
@jshaughn I was doing several tests and it works as expected. 

I've just added a couple of comments.

File: frontend/cypress/integration/common/graph_toolbar.ts [hunk 2]
```diff
@@ -96,15 +95,14 @@ Then('user sees default graph traffic menu', () => {
...
-    if (serverConfig.ambientEnabled) {
-      cy.get('input').should('have.length', 15);
-      cy.get('input#ambient').should('exist').should('be.checked');
-      cy.get('input#ambientWaypoint').should('exist').should('not.be.checked');
-      cy.get('input#ambientZtunnel').should('exist').should('not.be.checked');
-      cy.get('input#ambientTotal').should('exist').should('be.checked');
-    } else {
-      cy.get('input').should('have.length', 11);
-    }
+    // if we enable ambient in the test setup then include:
```
[the remark this answers is not in the document]
Comment (jshaughn) [reply frontend/cypress/integration/common/graph_toolbar.ts:98]: I didn't realize we had ambient support already, I'll try it!

**Review by josunect — APPROVED** [review]
LGTM!

Great improvement for Ambient

**Pull request merged**